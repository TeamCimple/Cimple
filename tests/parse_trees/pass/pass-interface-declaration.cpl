interface MyInterface {
	int getA();
	int getB();
};

int main() {
	return 0;
}
