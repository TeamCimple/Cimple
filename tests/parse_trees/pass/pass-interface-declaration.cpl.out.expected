INTERFACE(MyInterfaceFuncDecl(Name: Var(getB) ReturnType: int Parameters:  Body: CompoundStatement(Declarations:  StatementList: )) , FuncDecl(Name: Var(getA) ReturnType: int Parameters:  Body: CompoundStatement(Declarations:  StatementList: )) )FuncDecl(Name: Var(main) ReturnType: int Parameters:  Body: CompoundStatement(Declarations:  StatementList: RETURN(Int(0)))) 
